rdrf/rdrf/templates/rdrf_cdes/patient_edit.html

```diff
@@ -305,24 +306,20 @@
                                                                         {% endif %}
                                                                     </label>
                                                                     <div class="col-sm-9">
-                                                                        {% if not element|is_checkbox %}
+                                                                        {% if not element|is_checkbox and not element|is_radio %}
```

Comment:
Why are these ifs needed at all? Can't we just always use `element` and let these take care of setting the `form-control` CSS class?

https://github.com/eresearchqut/trrf/blob/ad581bb3acd6b0f2dc836ef910af6e5d5805944c/rdrf/rdrf/templates/rdrf_cdes/patient_edit.html#L124

Reply:
Indeed. Did not notice this.